modules/installation-vsphere-infrastructure.adoc

```diff
@@ -52,34 +52,32 @@ You must install the {product-title} cluster on a VMware vSphere version 7 insta
 [cols=2, options="header"]
 |===
 |Virtual environment product |Required version
-|VM hardware version | 15 or later
+|VMware virtual hardware | 15 or later
 |vSphere ESXi hosts | 7.0.2 or later
 |vCenter host   | 7.0.2 or later
 |===
 
 [IMPORTANT]
 ====
-Installing a cluster on VMware vSphere version 7.0.1 or earlier is now deprecated. These versions are still fully supported, but version 4.11 of {product-title} requires vSphere virtual hardware version 15 or later. Hardware version 15 is now the default for vSphere virtual machines in {product-title}. To update the hardware version for your vSphere nodes, see the "Updating hardware on nodes running in vSphere" article.
-
-If your vSphere nodes are below hardware version 15 or your VMware vSphere version is earlier than 6.7.3, upgrading from {product-title} 4.10 to {product-title} 4.11 is not available.
+Installing a cluster on VMware vSphere versions 7.0.0 and 7.0.1 is deprecated. These versions are still fully supported, but all vSphere 6.x versions are no longer supported. Version 4.12 of {product-title} requires VMware virtual hardware version 15 or later. To update the hardware version for your vSphere hosts, see the "Updating hardware on nodes running in vSphere" article in the _Updating clusters_ section.
```

Comment:
`vSphere hosts` should be `vSphere virtual machines`?

Reply:
> `vSphere hosts` should be `vSphere virtual machines`?

Yes, vSphere virtual machines, not hosts (hosts would implicitly refer to ESXi hosts).

Reply:
I've made this change. Thank you